app/helpers/snowcrash_helper.rb

```diff
@@ -40,4 +40,29 @@ def present(object, klass = nil)
     yield presenter if block_given?
     presenter
   end
+
+
+  def icon_for_model(model, icon_class, extra_class = nil)
```

Comment:
as a newbie, I think that a name that related this method to adding color to the icon would have helped me. Mey be something like `color_icon`,  `paint_icon`, `paint_icon_from_tag`, `colored_icon`...